content/cli/v6/commands/npm-unpublish.md

```diff
@@ -37,7 +37,7 @@ Even if a package version is unpublished, that specific name and
 version combination can never be reused. In order to publish the
 package again, a new version number must be used. If you unpublish the entire package, you may not publish any new versions of that package until 24 hours have passed.
 
-To learn more about how unpublish is treated on the npm registry, see our <a href="https://www.npmjs.com/policies/unpublish" target="_blank" rel="noopener noreferrer"> unpublish policies</a>. 
+To learn more about how unpublish is treated on the npm registry, see our <a href="https://docs.npmjs.com/policies/unpublish" target="_blank" rel="noopener noreferrer"> unpublish policies</a>. 
```

Comment:
I believe that this change will need to land in https://github.com/npm/cli/blob/v6/docs/content/cli-commands/npm-unpublish.md

(notice that this is on the v6 branch).

All of the CLI docs are vendored in.